.classpath

```diff
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="src" path="server_scripts"/>
 	<classpathentry excluding=".gitignore" kind="src" path="src"/>
-	<classpathentry kind="src" path="performanceanddevelopment"/>
```

Comment:
1. do we need to keep these two folders as cp are removed?
2. do we need to keep .classpath file as we switch build tool from Ant to Gradle?

Reply:
1. Yes, the code in there is still useful as a historical resource. I just removed it from the classpath to stop eclipse yelling at me about it
2. That'll happen in one of the last commits

Reply:
👍